### PR TITLE
change schema to october

### PR DESCRIPTION
**Pull request by James-Sangalli**
@colourful-land please resume work on return types from this PR and then we can update everything and merge

### CODE REVIEW

**Comment by colourful-land** [comment]
This issue intends to solve #225 but it's unfished yet

**Comment by RAZlEL** [comment]
access

**Comment by colourful-land** [comment]
@James-Sangalli I've done my changes to add `bitmask` to the `ethereum` element. Can you check if every example passes the new schema and if so, update every example to refer to the new schema? Thanks.

File: schema/data.xsd [hunk 1]
```diff
@@ -1,3 +1,117 @@
...
+                <xs:element name="int200" type="variable"/>
+                <xs:element name="int208" type="variable"/>
+                <xs:element name="int216" type="variable"/>
+                <xs:element name="int224" type="variable"/>
+                <xs:element name="int232" type="variable"/>
+                <xs:element name="int240" type="variable"/>
+                <xs:element name="int248" type="variable"/>
+                <xs:element name="int256" type="variable"/>
+                <xs:element name="string" type="variable"/>
+                <xs:element name="bytes" type="variable"/>
```
Comment (James-Sangalli) [inline schema/data.xsd]: Still need bytes1, even though it is the same as byte

**Comment by James-Sangalli** [comment]
> @James-Sangalli I've done my changes to add `bitmask` to the `ethereum` element. Can you check if every example passes the new schema and if so, update every example to refer to the new schema? Thanks.

@colourful-land schema validation fails on legacy tokenscripts,try it out in the intellij IDE as it will syntax highlight. 

Error for compound ZRX token: 

`make cZRX.canonicalized.xml
# XML Canonicalization
xmlstarlet c14n cZRX.xml  > cZRX.canonicalized.xml
# XML Validation
xmlstarlet val --xsd http://tokenscript.org/2019/05/tokenscript/tokenscript.xsd cZRX.canonicalized.xml || (mv cZRX.canonicalized.xml cZRX.canonicalized.xml.INVALID; xmllint --noout --schema http://tokenscript.org/2019/05/tokenscript/tokenscript.xsd cZRX.canonicalized.xml.INVALID)
cZRX.canonicalized.xml - invalid
http://tokenscript.org/2019/05/tokenscript/xhtml1-strict.xsd:33: element import: Schemas parser warning : Element '{http://www.w3.org/2001/XMLSchema}import': Skipping import of schema located at 'http://www.w3.org/2001/xml.xsd' for the namespace 'http://www.w3.org/XML/1998/namespace', since this namespace was already imported with the schema located at 'http://tokenscript.org/2019/05/tokenscript/xml.xsd'.
cZRX.canonicalized.xml.INVALID:1: element token: Schemas validity error : Element '{http://tokenscript.org/2019/10/tokenscript}token': No matching global declaration available for the validation root.
cZRX.canonicalized.xml.INVALID fails to validate
make: [cZRX.canonicalized.xml] Error 3 (ignored)`

**Comment by James-Sangalli** [comment]
@colourful-land might be good to add arrays to this Schema version as well?

File: schema/data.xsd (hunk 1 above)
Comment (colourful-land) [reply schema/data.xsd]: > @colourful-land might be good to add arrays to this Schema version as well?

No let's avoid adding it only to immediately change it in the next schema. Reason: ASN.1 compatibility desirable.

**Comment by colourful-land** [comment]
> > @James-Sangalli I've done my changes to add `bitmask` to the `ethereum` element. Can you check if every example passes the new schema and if so, update every example to refer to the new schema? Thanks.
> 
> @colourful-land schema validation fails on legacy tokenscripts,try it out in the intellij IDE as it will syntax highlight.
> 
> Error for compound ZRX token:
> 
> `$ make cZRX.canonicalized.xml cZRX.canonicalized.xml.INVALID:1: element token: Schemas validity error : Element '{http://tokenscript.org/2019/10/tokenscript}token': No matching global declaration available for the validation root. cZRX.canonicalized.xml.INVALID fails to validate make: [cZRX.canonicalized.xml] Error 3 (ignored)`

Look like you didn't change the namespace of your legacy TokenScript?

**Comment by James-Sangalli** [comment]
@colourful-land yup, it's working but the IDE doesn't like it mostly likely because the web path doesn't exist yet

**Comment by James-Sangalli** [comment]
@colourful-land have changed the TS files and removed the JS hack for compound e8 multipliers. Still WIP until we make the tokenscript schema for 2019/10 on the web

**Comment by James-Sangalli** [comment]
@colourful-land also need to add the return-type

**Comment by James-Sangalli** [comment]
> @colourful-land also need to add the bitmask

Apologies, see you already have this

**Comment by James-Sangalli** [comment]
@colourful-land ready to merge

**Comment by James-Sangalli** [comment]
@colourful-land iOS completed, just need to do android now

**Comment by James-Sangalli** [comment]
Android complete but needs review and iOS needs to fix issues with new schema

**Comment by James-Sangalli** [comment]
@colourful-land #252 is solved in 579a8d6129de3a5e32aefcdf15755b190136f21b

**Comment by James-Sangalli** [comment]
iOS problem fixed here: https://github.com/AlphaWallet/alpha-wallet-ios/pull/1495 once merged we are ready to go

**Pull request merged**